app/models/media.rb

```diff
@@ -43,14 +43,13 @@ def jsondata(context = nil)
   def data(context = nil)
     #TODO:: change the assumsion for a one Pender result
     em_pender = self.annotations('embed').last
-    embed = em_pender.embed unless em_pender.nil?
+    embed = JSON.parse(em_pender.embed) unless em_pender.nil?
     em_u = self.annotations('embed')
     context_id = context.nil? ? nil : context.id
     em_u.reverse.each do |obj|
       if obj.context_id.to_i == context_id.to_i
```

Comment:
@infojunkie  passing a nil context return all annotations not annotation with context nil
we should fix `annotations('embed', nil)` to return annotations with nil context only